contracts/adapters/TokenRegistry.sol

```diff
@@ -10,11 +10,12 @@ import "../../interfaces/IBZx.sol";
 
 contract TokenRegistry {
 
-    address public constant bZxContract = 0xD8Ee69652E4e4838f2531732a46d1f7F584F0b7f; // mainnet
+    //address public constant bZxContract = 0xD8Ee69652E4e4838f2531732a46d1f7F584F0b7f; // mainnet
```

Comment:
can you please add optimism addys but leave mainnet as default one ?

Reply:
updated